provisioning/device/src/polling_state_machine.ts

```diff
@@ -192,20 +185,15 @@ export class  PollingStateMachine extends EventEmitter {
     });
   }
 
-  register(registrationId: string, authorization: ProvisioningAuthentication, requestBody: any, forceRegistration: boolean, callback: (err?: Error, responseBody?: any, result?: any) => void): void {
-    debug('register called for registrationId "' + registrationId + '"');
-    this._fsm.handle('register', callback, registrationId, authorization, requestBody, forceRegistration);
+  register(request: RegistrationRequest, authorization: ProvisioningAuthentication, requestBody: any, callback: (err?: Error, responseBody?: any, result?: any) => void): void {
```

Comment:
>authorization [](start = 41, length = 13)

yeah seeing it here with the ProvisioningAuthentication type i think it'd be good to make a pass over the code and make sure we use authentication everywhere instead of authorization.